extension/src/experiments/model/collect.ts

```diff
@@ -312,19 +317,51 @@ const collectFromBranchesObject = (
   }
 }
 
+const getCommitDataFromOutput = (
+  output: string
+): CommitData & { hash: string } => {
+  const data: CommitData & { hash: string } = {
+    author: '',
+    date: '',
+    hash: '',
+    message: '',
+    tags: []
+  }
+  const [hash, author, date, refNamesWithKey] = output
+    .split('\n')
+    .filter(Boolean)
+  data.hash = hash
+  data.author = author
+  data.date = date
+
+  const message = output.match(/\nmessage:(.+)/s) || []
+  data.message = message[1] || ''
```

Comment:
I believe this is going to break if `message` is empty. It'll get an out of range index errror.

Reply:
I'll adjust the code accordingly! Though, I'm a little confused why the error would be thrown. Wouldn't `message[1]` just return undefined?

Reply:
You're right. I don't know where my previous assumption came from.

Reply:
> You're right. I don't know where my previous assumption came from.

No worries, I'll just undo my last commit 😄 I think other languages would have thrown that kind of error, like Python.